packages/build/src/plugins_core/blobs_upload/utils.ts

```diff
@@ -0,0 +1,61 @@
+import fs from 'node:fs'
+import fsPromises from 'node:fs/promises'
+import path from 'node:path'
+
+import { fdir } from 'fdir'
+
+const metadataPrefix = '$'
+const metadataSuffix = '.json'
+
+export const getBlobsDir = function ({ buildDir, publishDir }) {
+  const blobsPath = '.netlify/blobs/deploy'
+  const absoluteBlobsPath = path.resolve(buildDir, publishDir, blobsPath)
+  const blobsDir = path.normalize(absoluteBlobsPath)
```

Comment:
Do we need `path.normalize` here considering we run `path.resolve` just above?

Reply:
Probably not, and it was my lack of knowledge on Node.js 😅 
Updated in 6ff3ab833